securedrop/journalist.py

```diff
@@ -400,24 +418,27 @@ def _set_diceware_password(user, password):
     try:
         user.set_password(password)
     except PasswordError:
-        flash('You submitted a bad password! Password not changed.', 'error')
+        flash(gettext(
+            'You submitted a bad password! Password not changed.'), 'error')
         return
 
     try:
         db_session.commit()
     except Exception:
-        flash('There was an error, and the new password might not have been '
-              'saved correctly. To prevent you from getting locked '
-              'out of your account, you should reset your password again.',
-              'error')
+        flash(gettext(
+            'There was an error, and the new password might not have been '
+            'saved correctly. To prevent you from getting locked '
+            'out of your account, you should reset your password again.'),
+            'error')
         app.logger.error('Failed to update a valid password.')
         return
 
     # using Markup so the HTML isn't escaped
-    flash(Markup("<p>The password was successfully updated! Don't forget to "
-                 'save it in your KeePassX database. The new password is: '
-                 '<span><code>{}</code></span></p>'.format(password)),
-          'success')
+    flash(Markup("<p>" + gettext(
```

Comment:
This should be: 
```python
    flash(Markup(("<p>" + gettext(
        "The password was successfully updated! Don't forget to "
        "save it in your KeePassX database. The new password is: " 
        ' <span><code>{password}</code></span>') + '</p>').format(passworkd=password)),
        'success')
```

Because the HTML got unevenly split which could be confusing. Also, some languages don't use the colon that way and it would be unnatural to force them to.

Reply:
Note the gettext closing parenthesis is right after **password is:** so the string does not contain any html tag.

Reply:
Yeah, you're right, but my point still stands about about the colon. I guess we can leave it for now since I'm pretty sure non-English speakers are used to some oddities when using translated sites, should that be a problem.

Reply:
hum, you're right. It looks like chinese does not have a problem with that. Do you know of a language that does ? Maybe arabic  (@redshiftzero do you know ?). If you don't mind I'll leave it this way unless we have evidence it's wrong. I'm being lazy (of course, always am ;-). But also it means translators in Dutch, Chinese and Norwegian will have to update their translation if we change that (these are the three languages with active translators at the moment, not counting French of course). I'd like to make sure this is actually useful before we do the change and ask them that kind of effort.

Reply:
I can't think of one. Let's leave it for now.

Reply:
Not sure how Arabic handles colons tbqh - agree with leaving as is until we find a language that motivates changes here